scuba/config.py

```diff
@@ -517,7 +520,7 @@ def from_dict(
             return cls(
                 name=name,
                 script=script,
-                image=node.get("image"),
+                image=_get_str(node, "image"),
```

Comment:
This fix is actually independent from the `!reference` constructor addition, but only the latter is mentioned anywhere.

At a minimum, please update the commit message and/or PR description to call this out explicitly. The word "image" doesn't appear anywhere but the diff.

Bonus points if you put this (and the associated test addition) into their own commit. More bonus points if it is its own PR -- since this isn't really a GitLab "extension" to yaml, it's just a part of their schema that Scuba doesn't (yet) support.